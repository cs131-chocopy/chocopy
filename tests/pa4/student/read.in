12
1
2
3
4
55
123
456
888
8899
22
11451419
1919810
