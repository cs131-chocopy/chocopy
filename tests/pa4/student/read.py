def read()->int:
    x:int=0
    i:str=""
    j:int=0
    s:str=""
    s=input()
    for i in s:
        j=0
        while j<10:
            if i==char[j]:
                x=x*10+j
            j=j+1
    return x
char:[str]=None
a:[int]=None
n:int=0
i:int=0
t:int=0
char=["0","1","2","3","4","5","6","7","8","9"]
a=[]
n=read()
i=0
while i<n:
    t=read()
    a=a+[t]
    i=i+1
i=0
while i<n:
    a[i]=a[i]*2
    i=i+1
for i in a:
    print(i)
