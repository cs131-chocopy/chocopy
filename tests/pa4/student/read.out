2
4
6
8
110
246
912
1776
17798
44
22902838
3839620
